vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/HttpRequestSteps.java

```diff
@@ -58,9 +54,11 @@ public HttpRequestSteps(CookieManager<Cookie> cookieManager, HttpTestContext htt
     public void executeRequestUsingBrowserCookies()
     {
         CookieStore cookieStore = cookieManager.getCookiesAsHttpCookieStore();
+        List<org.apache.hc.client5.http.cookie.Cookie> cookies = cookieStore.getCookies();
         attachmentPublisher.publishAttachment(ATTACHMENT_TEMPLATE_PATH,
-                Map.of(TEMPLATE_COOKIES_KEY, cookieStore.getCookies()), "Browser cookies");
-        httpTestContext.putCookieStore(cookieStore);
```

Comment:
I think it makes sense to remove the method `putCookieStore` from `HttpTestContext`